<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>497</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>QChar常用功能</string>
  </property>
  <widget class="QGroupBox" name="groupBox_ButtosFunc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>其它测试和功能</string>
   </property>
   <widget class="QPushButton" name="pushButton_ClearText">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>155</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>清空文本框</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_QCharToLatin1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>155</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>与Latin1的转换</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_QCharCompare">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>155</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>QChar比较和替换</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_QCharToUTF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>155</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>与UTF-16的转换</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>341</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_result">
      <property name="title">
       <string>结果显示</string>
      </property>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>161</width>
         <height>281</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_ChkBox">
      <property name="title">
       <string>QChar特性判断</string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>141</width>
         <height>281</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="checkBox_isDigit">
          <property name="text">
           <string>isDigit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_isLetter">
          <property name="text">
           <string>isLetter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_isLetterOrNumber">
          <property name="text">
           <string>isLetterOrNumber</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_isUpper">
          <property name="text">
           <string>isUpper</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_isMark">
          <property name="text">
           <string>isMark</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_isLower">
          <property name="text">
           <string>isLower</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_isSymbol">
          <property name="text">
           <string>isSymbol</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_isPunct">
          <property name="text">
           <string>isPunct</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>输入字符串</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_String">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_QCharUnicode">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>每个字符的Unicode</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_QCharProperty">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>每个字符特性判断</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Char">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>72</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>输入单个字符</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
